src/components/PipelineRunDetailsView/utils/pipelinerun-utils.ts

```diff
@@ -0,0 +1,16 @@
+import { PipelineRunLabel } from '../../../consts/pipelinerun';
+import { PipelineRunKind } from '../../../types';
+
+export const getSourceUrl = (pipelineRun: PipelineRunKind): string => {
+  if (!pipelineRun) {
+    return undefined;
+  }
+
+  const gitUrl = pipelineRun.metadata?.annotations?.[
+    PipelineRunLabel.COMMIT_FULL_REPO_URL_ANNOTATION
+  ]
+    ? pipelineRun.metadata?.annotations?.[PipelineRunLabel.COMMIT_FULL_REPO_URL_ANNOTATION]
+    : (pipelineRun.spec?.params?.find((p) => p.name === 'git-url')?.value as string);
```

Comment:
Why not also look at this annotation and remove the query param:
`build.appstudio.openshift.io/repo: https://github.com/christianvogt/devfile-sample?rev=1e0f5587bb0a4986071ddae9a2d59834c3cf8432`

Reply:
Updated.